rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] After creating book and person, `rental is adding itself to the @book achieving the 'belongs-to' association`, So add these lines after initializing book and person.
 ```
    @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self
    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
```